src/stylesheets/components/global/task/task.scss

```diff
@@ -22,6 +22,7 @@
         
         .task__content--expanded {
             background: $task-content-background-light;
+			overflow: scroll;
```

Comment:
Cannot scroll overflow of subtasks in **dark mode** because of where this rule is added

Reply:
Agreed!Good catch!